README.md

```diff
@@ -1 +1,18 @@
-# dcf-dataservice
\ No newline at end of file
+# dcf-dataservice
+Copy data buckets from GDC to AWS and Google buckets. The jobs need to be run inside a setup kubeneste cluster.
+
+GDC has two different data storages: Data center and AWS backup bucket which are synchronized continuously with each other. Every one or two months, GDC releases new/updated/deleted data in manifest files, the goal is to deploy two separated jobs which copy the data to AWS, GOOGLE cloud respectively.
+
+## GDC AWS bucket to DCF AWS buckets
+Simplely using AWS API to sync in-AWS buckets. To boost up the performance, we deploy multiple-thread aws s3 copying.
```

Comment:
```suggestion
 Simply using AWS API to sync in-AWS buckets. To boost up the performance, we deploy multiple-thread aws s3 copying.
```

Reply:
fixed in d47a36c889171c98bbe27a87e89fb6d55d356f0f